hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/gas/TinybarValuesTest.java

```diff
@@ -27,46 +26,73 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
 class TinybarValuesTest {
     private static final ExchangeRate RATE_TO_USE =
```

Comment:
Please add a comment here pointing out this is where the magic number `7`, used throughout this test, comes from.  Otherwise it is somewhat mysterious.

Alternatively, perhaps `7` should be a named constant and used throughout _including_ in this line as `...newBuilder().hbarEquiv(2).centEquiv(2 * HBAR_CENT_RATIO)...`

Reply:
Done.